src/main/telemetry/crsf.c

```diff
@@ -431,7 +431,7 @@ void speedNegotiationProcess(uint32_t currentTime)
         crsfRxSendTelemetryData();
     } else {
         if (crsfSpeed.hasPendingReply) {
-            bool found = crsfSpeed.index < BAUD_COUNT ? true : false;
+            bool found = crsfSpeed.index < BAUD_COUNT && crsfRxUseNegotiatedBaud() ? true : false;
```

Comment:
Can you add parenthesis here to explicitly show precedence?

Reply:
Technically is correct:

![image](https://user-images.githubusercontent.com/8344830/158995179-60684480-0559-44ec-ab29-d2a2b2d1472e.png)

Reply:
Added paranthesis